guest/common/include/s2e/s2e.h

```diff
@@ -336,20 +336,12 @@ static inline void s2e_disable_forking(void)
 /// \brief Forks the given number of times without adding constraints
 ///
 /// \param[in] count The number of times to fork
-/// \param[in] name Label
-/// \return ???
 ///
-static inline int s2e_fork(int count, const char *name) {
-    unsigned result = 0;
-    __s2e_touch_string(name);
+static inline void s2e_fork(int count) {
```

Comment:
Perhaps returning the index of the path would be useful here (e.g., if you fork n times, first path will be 0, last path n-1).